operator/v1/types_kubeapiserver.go

```diff
@@ -33,6 +33,23 @@ type KubeAPIServerSpec struct {
 
 type KubeAPIServerStatus struct {
 	StaticPodOperatorStatus `json:",inline"`
+
+	// serviceAccountIssuers tracks history of used service account issuers.
+	// The item without expiration time represents the currently used service account issuer.
+	// The other items represents service account issuers that were used previously and are still being trusted.
+	// see: https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/#service-account-token-volume-projection
+	// +optional
+	ServiceAccountIssuers []ServiceAccountIssuerStatus `json:"serviceAccountIssuers,omitempty"`
+}
+
+type ServiceAccountIssuerStatus struct {
+	// name is the name of the service account issuer
+	Name string `json:"string"`
```

Comment:
Please add the `+kubebuilder:validation:Required` tag to this

What is the expected format of the issuer of the service account?

Reply:
```
	// serviceAccountIssuer is the identifier of the bound service account token
	// issuer.
	// The default is https://kubernetes.default.svc
	// WARNING: Updating this field will result in the invalidation of
	// all bound tokens with the previous issuer value. Unless the
	// holder of a bound token has explicit support for a change in
	// issuer, they will not request a new bound token until pod
	// restart or until their existing token exceeds 80% of its
	// duration.
	// +optional
	ServiceAccountIssuer string `json:"serviceAccountIssuer"`
```

It is just a copy of this API field, which also does not have the format specified.
The format is an URL if I understand this right. This is a status field, so not a user-facing API IMHO.